File: C3/C3 08 AS 03 Loops/08.cpp
//08. Write a C++ program to calculate factorial of a number.
#include<iostream>
using namespace std;
int main()
{
    int n, fact=1;
    cout<< "Enter a number: ";
    cin>>n;
    cout<< "Factorial of " << n << " is: ";
    for(n;n;n--)
        fact*=n;
    cout<< fact;
}

File: Python/PY 25 AS 11 more on while/08.py
#   08. Write a python script to calculate sum of digits of a given number.
n = int(input("Enter a number: "));
print("Sum of digit of %d is: "%(n),end="");
sum = 0;
while n:
    sum = sum + (n%10);
    n//=10;
print(sum);